src/components/Slider/Slider.css

```diff
@@ -0,0 +1,135 @@
+/* Styles for Swiper cards */
+
+.swiper-wrapper img {
+  height: 60px;
+  width: 75px;
+  object-fit: cover;
+}
+
+.record--artist,
+.record--album {
+  font-family: "Libre Baskerville", sans-serif;
+}
+
+.record--artist {
+  font-size: 0.75rem;
+  text-align: left;
+  margin-top: 0.5rem;
+}
+
+.record--album {
+  font-size: 0.6rem;
+  opacity: 0.6;
+  text-align: left;
+  margin-top: 0.25rem;
+}
+
+/* Swiper button and navigation container styles */
+.swiper-functions {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 0.5rem;
+}
+
+.custom-swiper-navigation {
+  display: flex;
+}
+
+.swiper-pagination-fraction,
+.swiper-pagination-custom,
+.swiper-horizontal > .swiper-pagination-bullets,
+.swiper-pagination-bullets.swiper-pagination-horizontal {
+  bottom: auto !important;
+  left: auto !important;
+  top: auto !important;
+  width: auto !important;
+  position: relative !important;
```

Comment:
Not important to change, just pointing out what I like to do. 

When i use an important tag i like to give a small comment why. So for this reason it's because we are overriding the swiper styles.

Reply:
completed - added comments